tests/bootstrap.php

```diff
@@ -1,3 +1,7 @@
 <?php
 
 require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+if (class_exists('PHPUnit_Framework_TestCase') && !class_exists('PHPUnit\Framework\TestCase')) {
+    class_alias( 'PHPUnit_Framework_TestCase', 'PHPUnit\Framework\TestCase');
+}
```

Comment:
Can you add a new line at the end of this file?

Reply:
Done.